karma.conf.js

```diff
@@ -4,7 +4,9 @@ module.exports = function( config ) {
     var isJenkins = !!process.env.JENKINS_HOME;
     var serverPort = process.env.KARMA_PORT || 9876;
     var testReporters = [
-        'progress',
+        'spec',
+        // 'progress',
+        // 'dots',
```

Comment:
I will. progress is pretty useless unless you care about the number of tests before the runner is done which we should not. "dots" is actually really useful when debugging because it outputs console logs to stdio.